src/Sentry.Unity.Native/SentryNativeBridge.cs

```diff
@@ -12,13 +12,11 @@ namespace Sentry.Unity.Native;
 /// P/Invoke to `sentry-native` functions.
 /// </summary>
 /// <see href="https://github.com/getsentry/sentry-native"/>
-public static class SentryNativeBridge
+internal static class SentryNativeBridge
 {
-    public static bool CrashedLastRun;
-
-    public static bool Init(SentryUnityOptions options, ISentryUnityInfo sentryUnityInfo)
+    public static bool Init(SentryUnityOptions options, ISentryUnityInfo? sentryUnityInfo)
     {
-        _isLinux = sentryUnityInfo.IsLinux();
+        _isLinux = sentryUnityInfo?.IsLinux() ?? false;
```

Comment:
since this is all internal code, should we expect `null` `ISentryUnityInfo`? 
Seems to me the caller should either have the instance or avoid getting an insance of this altogether?

If we have that null here, we'd still go ahead initializing everything below. Is that creating a weird scenario where instead we should have avoided `Init` being successful?

Reply:
Reworked the PlatformServices.UnityInfo handling by pulling this into the options.